### PR TITLE
Oop UI

### PR DESCRIPTION
**Pull request by boniface193**
created app.rb file that will serve as console app entry-point.
main.rb define the entry point.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  def student_teacher
+    print 'Do you want to create a student (1)  or a teacher (2):'
+    name = gets.chomp
+
+    if name.to_i > 2
+      puts 'invalid option'
+    else
+      add_person
+    end
+  end
```
Comment (ridwanediallo) [inline app.rb]: - [ ] Well done implementing the features in your app console 👏. However, I have a few suggestions. 
 - Please let's make sure that when creating a person whether they are a student or a teacher not only get the `name` but their `age` too. And if the person is a student we should ask for permission or if they are a teacher we should ask for their specialisation as in the [demo video](https://www.youtube.com/watch?v=vkkgrhD6aXQ). 👌 😊

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @boniface193 , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Your app is working correctly. ✔


## Required Changes ♻️

- [ ] Please I would suggest you to remove the output from the person class because I think you don't need it now. 

### piece of code :point_down: 
Screenshot from 2023-10-13 12-46-21


### output on the console :point_down: 
Screenshot from 2023-10-13 13-03-52






## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mukhammadrizooff** [comment]
Hi @boniface193, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that linters failed, it should be pass 👍🏼 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @boniface193👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Good job with the console app. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**